packages/@o3r/training-sdk/typedoc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we generate typedoc in from this package ? We should probably avoid it if this is the case

Reply:
this file was automatically generated and is also present in the showcase-sdk, should we remove this?